rice-framework/krad-sampleapp/web/pom.xml

```diff
@@ -38,7 +38,7 @@
     <jmeter.excludeJmeterTestFile>**/KRAD.jmx</jmeter.excludeJmeterTestFile>
     <kuali.testjar.phase>none</kuali.testjar.phase>
     <kuali.generate.reports>false</kuali.generate.reports>
-    <webapp.working.dir>${project.build.directory}/${project.artifactId}-${project.version}</webapp.working.dir>
+    <webapp.working.dir>${project.build.directory}/${build.finalName}</webapp.working.dir>
```

Comment:
Why did we need to replace ${project.artifactId}-${project.version} with ${build.finalName} and where will that be declared?